tekton/resources/nightly-release/overlays/operator/kustomization.yaml

```diff
@@ -15,4 +15,4 @@ patchesJson6902:
       name: nightly
     path: trigger.yaml
 resources:
-  - github.com/tektoncd/operator/tekton/?ref=main
\ No newline at end of file
+  - github.com/nikhil-thomas/tektoncd-operator/tekton/overlays/nightly-releases?ref=main
```

Comment:
This should still refer `tektoncd/operator` though 😝

Reply:
i remembered to change the `ref=main` but missed it in the url... 😛 

fixed 🧑‍💻 👍